Classes/Utility/Dimension.php

```diff
@@ -68,7 +70,7 @@ public static function extractHeight($str)
      * @param string $str
      * @return string
      */
-    public static function extractUnit($str)
+    public static function extractUnit($str): \string
```

Comment:
"Fatal error: Scalar type declaration 'string' must be unqualified"
-> should be ": string" instead of ": \string"